app/models/concerns/specimen_role.rb

```diff
@@ -7,6 +7,11 @@ module SpecimenRole
                            allow_blank: true,
                            message: "is not within valid options"
 
+    def specimen_role_description(id)
+      specimen_role_id = id || ""
+      self.class.specimen_role_description(specimen_role_id)
+    end
```

Comment:
```suggestion
    def specimen_role_description
      self.class.specimen_role_description(specimen_role)
    end
```

We don't need to pass the specimen role id to the method, we already have access to the `#specimen_role` attribute. Also the class method will already check if the value is nil.

This allows us to use `sample.specimen_role_description` and `batch.specimen_role_description` directly :raised_hands:

Reply:
Also keep in mind that changes were made to this concern in #1366 (that should be merged before this PR)